docs/beds/vibradorm.md

```
@@ -1,86 +1,184 @@
+| `MEM1`..`MEM6` | `0x0E` `0x0F` `0x0C` `0x1A` `0x1B` `0x1C` | Memory | memory slot recall |
+
+### CBI command set (2-byte BE word `[msb(toggle|cmd), lsb, …payload]` on `CBI` 0x1550)
+
+| Command | Hex (base) | Purpose | Payload |
+|---------|------------|---------|---------|
+| `CMD_STORE` | `0x0D` | store-to-memory arming | none |
+| `MEM1`..`MEM6` | `0x0E` `0x0F` `0x0C` `0x1A` `0x1B` `0x1C` | memory recall | none |
+| `CMD_DIM` | `0x11` | floor light (CBI variant) | `[level, timer_min]` |
+| `CMD_COLOR` | `0x77` (`0x1077` for XT box) | mood light base | varies by subcmd |
+| `CMD_VxEFF` | `0x30` (`0x1030` for XT box) | VRT settings | `[effect, speed, zone1, zone2, 0, 0, 0, timer]` |
+| `CMD_VRT` | `0x34` (`0x1034` for XT box) | VRT on/off | `[0|1]` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)